htdocs/core/lib/functions2.lib.php

```diff
@@ -1502,7 +1502,7 @@ function dol_print_reduction($reduction,$langs)
     }
     else
     {
-        $string = price($reduction).'%';
+        $string = $reduction.'%';
```

Comment:
The goal of function price here is to format the number into language of user.
So number 1.23 will be output   1,23   for france.
Old code looks good to me